lte/gateway/c/core/oai/test/ngap/util_ngap_amf_nas_procedures.cpp

```diff
@@ -207,6 +207,7 @@ bool generator_ngap_pdusession_resource_setup_req(bstring& stream) {
   return true;
 }
 
+#if 0 /*  CLEAN UP NOT NEEDED */
```

Comment:
why is that code just commented out? Wouldn't cleanup be better?

Reply:
Code cleaned up..